ddl/placement/bundle.go

```diff
@@ -376,6 +376,25 @@ func (b *Bundle) Reset(newID int64) *Bundle {
 	return b
 }
 
+// Append is used to append a slice of partition ids to use the default table-level rules.
+func (b *Bundle) Append(newIDs []int64) *Bundle {
```

Comment:
Let us make it the new `reset`. You need to reset `b.ID`, `rule.GroupID`, `b.Override`, and `b.Index`.